understanding-ibc-denoms/index.md

```diff
@@ -0,0 +1,191 @@
+```
+
+Response:
+
+```bash
+denom_trace:
+  base_denom: moon
+  path: transfer/channel-14
+```
+
+From this command, you now know that there is an IBC port `transfer` and channel `channel-14`. But to know the IBC light client behind the port and channel, you need to perform another query.
+
+Why is it called a light client? Because it is a light client of the _other_ chain, keeping track of its blockhashes. The `ibc channel client-state transfer` command  explains the details of the denom path.
+
+```bash
+gaiad query ibc channel client-state transfer channel-14 --node https://rpc.cosmos.network:443
+```
+
+Response
+
+```bash
+client_id: 07-tendermint-18
+client_state:
+  '@type': /ibc.lightclients.tendermint.v1.ClientState
+  allow_update_after_expiry: true
+  allow_update_after_misbehaviour: true
+  chain_id: mars
+  frozen_height:
+    revision_height: "0"
+    revision_number: "0"
+  latest_height:
+    revision_height: "2207"
+    revision_number: "0"
+  max_clock_drift: 600s
+  proof_specs:
+  - inner_spec:
+      child_order:
+      - 0
+      - 1
+      child_size: 33
+      empty_child: null
+      hash: SHA256
+      max_prefix_length: 12
+      min_prefix_length: 4
+    leaf_spec:
+      hash: SHA256
+      length: VAR_PROTO
+      prefix: AA==
+      prehash_key: NO_HASH
+      prehash_value: SHA256
+    max_depth: 0
+    min_depth: 0
+  - inner_spec:
+      child_order:
+      - 0
+      - 1
+      child_size: 32
+      empty_child: null
+      hash: SHA256
+      max_prefix_length: 1
+      min_prefix_length: 1
+    leaf_spec:
+      hash: SHA256
+      length: VAR_PROTO
+      prefix: AA==
+      prehash_key: NO_HASH
+      prehash_value: SHA256
+    max_depth: 0
+    min_depth: 0
+  trust_level:
+    denominator: "3"
+    numerator: "1"
+  trusting_period: 1209600s
+  unbonding_period: 1814400s
+  upgrade_path:
+  - upgrade
+  - upgradedIBCState
+```
+
+That's a lot of information, but it doesn't answer the question: how do you know if this IBC client can be relied upon?
+
+### The chain ID and The client ID
+
+Anybody can start a chain with the same chain ID, but the IBC client ID is generated by the [Cosmos SDK IBC Keeper module](https://github.com/cosmos/ibc-go/blob/e012a4af5614f8774bcb595962012455667db2cf/modules/core/02-client/keeper/keeper.go#L56) (ICS-02 does not specify a standard for IBC client IDs). A Chain Name Service and the not-so-decentralized Github chain-registrar repo can verify the combination of the two. Both the Chain Name Service and the chain-registrar repo are under development.
+
+### Ensure the IBC client isn't expired
+
+In the event that Tendermint consensus fails (if >1/3 of validators produce a conflicting block), _and_ proof of this consensus failure is submitted on chain, the IBC client becomes frozen with a `frozen_height` that is nonzero. In the above example this is not the case.
```

Comment:
```suggestion
In the event that Tendermint consensus fails (if >1/3 of validators produce a conflicting block), _and_ proof of this consensus failure is submitted on-chain, the IBC client becomes frozen with a `frozen_height` that is nonzero. In the previous example, the output of `gaiad query ibc channel client-state` confirms the client status and you know the IBC client is not expired. 
```

Reply:
I went digging to try to understand what "this" (a favorite of ours, the dreaded "unclear reference" we all love) is referring to... I _think_ that we're talking about the client status?

Reply:
You are correct 👍